vmhost/vmhooks/baseOps.go

```diff
@@ -2086,7 +2107,7 @@ func (context *VMHooksImpl) GetCurrentESDTNFTNonce(
 	}
 
 	key := []byte(core.ProtectedKeyPrefix + core.ESDTNFTLatestNonceIdentifier + string(tokenID))
-	data, _, err := storage.GetStorageFromAddress(destination, key)
+	data, _, _, err := storage.GetStorageFromAddress(destination, key)
```

Comment:
is this ok?
On v1.4 we have:
```GO
data, trieDepth, _, err := storage.GetStorageFromAddress(destination, key)
if vmhost.WithFault(err, context, runtime.BaseOpsErrorShouldFailExecution()) {
    return 0
}

err = storage.UseGasForStorageLoad(getCurrentESDTNFTNonceName, int64(trieDepth), metering.GasSchedule().BaseOpsAPICost.StorageLoad, false)
```

Reply:
It was not good; excellent catch. Fixed to behave as v1.4.